elliott

```diff
@@ -803,33 +842,78 @@ Example to look for bugs that have been moved back to MODIFIED
         # No bugs were provided
         raise click.BadParameter("If not using --auto then one or more --id's must be provided")
 
+    if len(id) == 0 and advisory is None and default_advisory_type is None:
+        # error, no bugs, advisory, or default selected
+        raise click.BadParameter("No input provided: Must use one of --id, --advisory, or --use-default-advisory")
+
     # Load bugzilla infomation and get a reference to the api
     runtime.initialize()
     bz_data = runtime.gitdata.load_data(key='bugzilla').data
     bzapi = elliottlib.bzutil.get_bzapi(bz_data)
-
     changed_bug_count = 0
     attached_bugs = []
 
+    # if --use-default-advisory is set we will override a user
+    # provided advisory
+    if default_advisory_type is not None:
+        default_advisory = runtime.group_config.advisories.get(default_advisory_type, None)
+        if default_advisory is not None:
+            advisory = default_advisory
+            green_prefix("Default advisory detected: ")
+            click.echo(advisory)
+        else:
+            red_prefix("No value defined for default advisory:")
+            click.echo(" The key `advisories.{}` is not defined for group {} in group.yml".format(
+                default_advisory_type, runtime.group))
+            exit(1)
+
+    raw_bug_list = []
     if auto:
+        click.echo("Fetching Erratum(errata_id={})".format(advisory))
         e = Erratum(errata_id=advisory)
-        attached_bugs = [bzapi.getbug(i) for i in e.errata_bugs]
+        raw_bug_list = e.errata_bugs
     else:
-        attached_bugs = [bzapi.getbug(i) for i in cli_opts.id_convert(id)]
-
+        click.echo("Bypassed fetching erratum, using provided advisory: {}".format(advisory))
```

Comment:
Shouldn't this be "Using provided BZ#s" ?

Reply:
Yes.

Reply:
Fixed